rpcs3/Emu/Memory/vm.cpp

```diff
@@ -515,7 +515,7 @@ namespace vm
 	bool check_addr(u32 addr, u32 size, u8 flags)
 	{
 		// Overflow checking
-		if (addr + size < addr && (addr + size) != 0)
+		if (addr > 0 - size)
```

Comment:
Seems to fail if size==0 (but should return true).

Reply:
It was not working well before it, [addr=0, size=0] would result in scanning all pages, but sure it can be fixed.